[IMP] Expenses: making targeted edits for v 15 (#4701)
odoo/documentation: content/applications/finance/expenses.rst

   :alt: Attach a receipt after saving the record.

Click the new :guilabel:`Attach Receipt` button, and a file explorer appears. Navigate to the
receipt to be attached, and click :guilabel:`Open`. The new receipt is recorded in the chatter, and
the number of receipts will appear next to the :guilabel:`📎(paperclip)` icon beneath the expense
form. More than one receipt can be attached to an individual expense, as needed. The number of
receipts attached to the expense will be noted on the paperclip icon.

.. image:: expenses/receipt-icon.png
   :align: center
   :alt: View receipts in the chatter and see how many receipts are attached byu looking at the
         paperclip icon in the chatter beneath the record.

Create new expenses from a scanned receipt
------------------------------------------

Rather than manually inputting all of the information for an expense, expenses can be created
automatically by scanning a PDF receipt.

> First, in the main *Expenses* app dashboard view (this view can also be accessed from

Reviewer: Yea menuselection could work or we could mimic what we did for 16 and do a gui here.

   :align: center
   :alt: Add more expenses to the report before submitting.

When all edits have been completed, click :guilabel:`Save`. Once the expense report is saved, an

> :guilabel:`Expenes` smart button appears in the top right, listing the number of expenses on the

Reviewer: ```suggestion
:guilabel:`Expenes` smart button appears in the top-right corner, listing the number of expenses on the
```

   :align: center
   :alt: Send messages in the chatter.

Team managers can easily view all the expense reports to approve for their team members. While in
the :guilabel:`Reports to Approve` view, click on :guilabel:`Filters`, then click :guilabel:`My

> Team`. This presents all the reports to approve for the manager's team. Manager's can easily view
> *all* reports for their team. Simply click the X to remove the :guilabel:`To Approve` filter,
> leaving only the :guilabel:`My Team` filter active.

Reviewer: suggestion to join the sentences and remove the **X**
```suggestion
Team`. This presents all the reports to approve for the manager's team. Manager's can easily view
*all* reports for their team by simply clicking the removing the :guilabel:`To Approve` filter in
the :guilabel:`Search...` bar,leaving only the :guilabel:`My Team` filter active.
```

.. image:: expenses/my-team-filter.png
   :align: center
   :alt: Select the My Team filter.

.. note::
   If more information is needed, such as a receipt is missing, communication is easy from the
   chatter. In an individual report, simply click :guilabel:`Send Message`, then type in a message
   in the text box that appears, tagging the proper person (if needed), and post it to the chatter
   by clicking :guilabel:`Send`. The message is posted in the chatter, and the person tagged will be
   notified via email of the message, as well as anyone following.

   Note that only followers can be tagged in a message. To tag someone who is not following, add
   them as a follower first. Click the :guilabel:`👤 (person)` icon, then click :guilabel:`Add
   Followers`, then enter their name in the :guilabel:`Recipients` field, and finally, click

>    :guilabel:`Add Followers`.

Reviewer: This whole section reads more as a **tip**, actually.
Also would start it like this:
`If more information is needed, such as a missing receipt, communication is  easy from the`

================================

If expenses are tracked on customer projects, expenses can be automatically charged back to the
customer. This is done by creating an expense, referencing the :abbr:`SO (Sales Order)` the expense
should be added to, and then create the expense report. Next, managers approve the expense report,
and the accounting department posts the journal entries. Finally, once the expense report is posted
to a journal, the expense appears on the :abbr:`SO (Sales Order)` that was referenced. The sales

> order can then be invoiced, thus invoicing the customer for the expense.

Reviewer: Reads clunky. Can you give this section another look?

- :guilabel:`At cost`: Expense product will invoice expenses at their real cost.
- :guilabel:`At sales price`: Expense product will invoice the price set on the sale order.

Create an expense and expense report
------------------------------------

First, when :ref:`creating a new expense <expenses/new>`, the correct information needs to be
entered in order to re-invoice a customer. Select the sales order the expense will appear on in the
:guilabel:`Customer to Reinvoice` section, from the drop-down menu. Next, select the

> :guilabel:`Analytic Account` the expense will be posted to. After the expense(s) are created, the

Reviewer: We introduce the instruction using expense (singular), so grammar should match accordingly.
```suggestion
:guilabel:`Analytic Account` the expense will be posted to. After the expense(s) is created, the
```

:guilabel:`Approve` or :guilabel:`Refuse`.

The accounting department is typically responsible for :ref:`posting journal entries
<expenses/post>`. Once an expense report is approved, it can then be posted. The :abbr:`SO (Sales

> Order)` is **only** updated *after the journal entries are posted*, and the expenses now appear on

Reviewer: mistmatch between singular and plural in this section
```suggestion
Order)` is **only** updated *after the journal entries are posted*, and the expense(s) now appear on
```
